demo/src/main/java/com/google/android/fhir/demo/data/DownloadWorkManagerImpl.kt

```diff
@@ -28,25 +32,28 @@ import org.hl7.fhir.r4.model.Reference
 import org.hl7.fhir.r4.model.Resource
 import org.hl7.fhir.r4.model.ResourceType
 
-class DownloadWorkManagerImpl : DownloadWorkManager {
+class DownloadWorkManagerImpl(private val dataStore: DemoDataStore) : DownloadWorkManager {
```

Comment:
maybe this name:

```suggestion
class TimestampBasedDownloadWorkManagerImpl(private val dataStore: DemoDataStore) : DownloadWorkManager {
```

?

I'll let you decide :)

Reply:
done, renamed class to `TimestampBasedDownloadWorkManagerImpl`